content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,237 @@
-=================================
-How to sell in foreign currencies
-=================================
+==================
+Foreign currencies
+==================
+
+With Odoo, pricelists can be used to manage prices in a number of foreign currencies. Specifically,
+Odoo has the ability to work with 167 total currencies. In order to use multiple currencies in Odoo
+*Sales*, the *Accounting* application **must** be installed.
+
+Settings
+========
 
-Pricelists can also be used to manage prices in foreign currencies.
+Once the *Accounting* app is installed, foreign currencies can be viewed, edited, and added to the
+database. To do that, navigate to :menuselection:`Accounting app --> Configuration --> Settings`,
```

Comment:
Would just be mindful of singular/plural use for the demonstrative pronoun here, as it's currently written there's a mismatch (i.e. does *that* mean add, edit, or view in relation to the sentence before it? Or should it actually be *those*?). Imo it's not needed anw.
```suggestion
database. Navigate to :menuselection:`Accounting app --> Configuration --> Settings`,
```